Adds the supported metric types. (#6754)
opensearch-project/documentation-website: _monitoring-your-cluster/metrics/getting-started.md

2. **Exporters:** Exporters are responsible for persisting the data. OpenTelemetry provides several out-of-the-box exporters. OpenSearch supports the following exporters:
    - `LoggingMetricExporter`: Exports metrics to a log file, generating a separate file in the logs directory `_otel_metrics.log`. Default is `telemetry.otel.metrics.exporter.class=io.opentelemetry.exporter.logging.LoggingMetricExporter`.
    - `OtlpGrpcMetricExporter`: Exports spans through gRPC. To use this exporter, you need to install the `otel-collector` on the node. By default, it writes to the http://localhost:4317/ endpoint. To use this exporter, set the following static setting: `telemetry.otel.metrics.exporter.class=io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter`.
  
### Supported metric types

The metrics framework feature supports the following metric types:

> 1. **Counters:** Counters are continuous and synchronous meters used to track the frequency of events over time. Counters can only be incremented with positive values making them ideal for measuring the number monitoring occurrences such as errors, processed or received bytes, and total requests.

Reviewer: ```suggestion
1. **Counters:** Counters are continuous and synchronous meters used to track the frequency of events over time. Counters can only be incremented with positive values, making them ideal for measuring the number of monitoring occurrences such as errors, processed or received bytes, and total requests.
```

> 2. **UpDown counters:**  UpDown counters can be incremented with positive values or decremented with negative values. UpDown counters are well-suited for tracking metrics like open connections, active requests, and other fluctuating quantities.

Reviewer: ```suggestion
2. **UpDown counters:** UpDown counters can be incremented with positive values or decremented with negative values. UpDown counters are well suited for tracking metrics like open connections, active requests, and other fluctuating quantities.
```

> 3. **Histograms:** Histograms are valuable tools for visualizing the distribution of continuous data. Histograms offer insights into the central tendency, spread, skewness, and any potential outliers that might exist inside your metrics Patterns such as normal distribution, skewed distribution, or bimodal distribution can be readily identified, making histograms ideal for analyzing latency metrics and assessing percentiles.

Reviewer: ```suggestion
3. **Histograms:** Histograms are valuable tools for visualizing the distribution of continuous data. Histograms offer insight into the central tendency, spread, skewness, and potential outliers that might exist in your metrics. Patterns such as normal distribution, skewed distribution, or bimodal distribution can be readily identified, making histograms ideal for analyzing latency metrics and assessing percentiles.
```
